_component_design/buttons.md

```diff
@@ -14,203 +14,193 @@ last-modified: 2017-08-17
-
--   Follows the same styles and states as the button weight which it extends
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
--   Attempt to limit to a maximum of 8 items. If more than 8 items are needed,
-    consider splitting a subset of actions into another location on the page.
+- Use the primary action button to prompt the user to take action.
+- Use only one primary button per page.
+- Simple buttons, dropdown buttons, and split dropdown buttons all share the same properties for state changes.
 
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Dropdown buttons"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-dropdowns-actions.svg)
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Button spacing specs"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-primary.svg){:width="100%"}
 {% endfigure %}
 {% endcolumn %}
 </div>
 
-# Processing state (Loading)
-When any button that submits an asynchronous api call is pressed the
-"processing" class/state is applied to the button. This state triggers the
-loading animation to be applied to the button.
-
+### Secondary buttons - simple
+{: .hxSubSectionTitle}
 <div class="hxRow">
-{% column left:"hxCol" %}
-- Details on loading animation specifics can be found on the <a href="{{site.baseurl}}/components/loading.html">loading page</a>.
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+- The secondary button can be used alone or in conjuction with a primary button or a tertiary button.
+- Use Secondary Buttons when there are multiple buttons on the same screen.
+- Simple buttons, dropdown buttons and split dropdown buttons  all share the same properties for state changes.
+
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Processing state across button weights"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-loading.svg)
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Button spacing specs"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-secondary.svg){:width="100%"}
 {% endfigure %}
 {% endcolumn %}
 </div>
 
-# Button Specs
+### Tertiary buttons - simple
+{: .hxSubSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-List of associated css values used across the three button sizes.
+- The tertiary button can be used alone, or in conjuction with other buttons. 
+- Use alone when you want to deemphasize the action, or when you have a list of actions.
+- Use in conjunction with a primary or secondary button when there are two options, but you want to de-emphasize one of them. For example, in a dialog box with two options, use “Submit” (Primary) and “Cancel” (Tertiary).
```

Comment:
Lowercase for `primary` and `tertiary` at the end

Reply:
updated.